dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard_progress_v2.feature

```diff
@@ -33,6 +33,80 @@ Scenario: Teacher can open and close Icon Key and details
   And I click selector "#ui-close-dialog"
   And element "h3:contains(Progress Tracking Icon Key)" is hidden
 
+Scenario: Teacher can open and close lessons and see level data cells
+  Given I create an authorized teacher-associated student named "Sally"
+  Given I am assigned to unit "allthethings"
+
+  When I sign in as "Teacher_Sally" and go home
+  And I get levelbuilder access
+  When I click selector "a:contains(Untitled Section)" once I see it to load a new page
+  And I wait until element "#uitest-teacher-dashboard-nav" is visible
+  And check that the URL contains "/teacher_dashboard/sections/"
+  And I wait until element "#uitest-course-dropdown" is visible
+  Then I append "/?enableExperiments=section_progress_v2" to the URL
+
+  # toggle to V2 progress view
+  Then I click selector "#ui-test-toggle-progress-view"
+  And element "#ui-test-progress-table-v2" is visible 
```

Comment:
I recognize that a lot of these steps are repeated... it is just kind of the process to getting to the V2 view under the current situation.  Once we move permanently over to V2 or "default to V2" many of these steps will be cut down... 

I could also put these steps into their own `steps.rb` function, but that feels a bit overkill for just this test file...